webodm.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+platform="Linux" # Assumed
+uname=`uname`
```

Comment:
I've found that `$(...)` is less error-prone than backticks (and possible to write in Markdown!).